spark-3.5.yaml

```diff
@@ -53,14 +54,15 @@ data:
       8: "java-1.8-openjdk"
       11: "java-11-openjdk"
       17: "java-17-openjdk"
+      21: "java-21-openjdk"
 
 subpackages:
   - range: openjdk-versions
     name: ${{package.name}}-openjdk-${{range.key}}
     dependencies:
-      # Both `python` and `openjdk-{11,17}-default-jvm` are required for running Spark,
-      # but will be added in the image, since upstream supports different Python versions
-      # and Java versions. So we don't need to specify them here to avoid conflicts.
```

Comment:
this comment doesn't make sense any more because prior PR forces inclusion of openjdk